docs/reference/api-overview.md

```diff
@@ -65,9 +65,13 @@ Currently, there are several API groups in use:
 *  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION`, and use `apiVersion: $GROUP_NAME/$VERSION`
    (for example, `apiVersion: batch/v1`).  Full list of supported API groups can be seen in [Kubernetes API reference](/docs/reference/).
 
-There is a supported path to extending the API:
-* [Third Party Resources](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/extending-api.md)
-   are for users with very basic CRUD needs.
+There are two supported paths to extending the API with [custom resources](/docs/concepts/extensions/custom-resources/):
```

Comment:
:+1: would be down----I'm guessing this might happen after 1.7 release? we can sync on it once you're less busy. I'll add it to my TODOs so I don't forget.